Cubical/Data/Nat/Base.agda

```diff
@@ -7,6 +7,16 @@ open import Agda.Builtin.Nat public
   using (zero; suc; _+_; _*_)
   renaming (Nat to ℕ)
 
+open import Agda.Builtin.FromNat public
+  renaming (Number to HasFromNat)
+open import Agda.Builtin.FromNeg public
+  renaming (Negative to HasFromNeg)
+open import Cubical.Data.Unit.Base public
+
+instance
+  fromNatℕ : HasFromNat ℕ
+  fromNatℕ = record { Constraint = λ _ → Unit ; fromNat = λ n → n }
```

Comment:
Also, what is this?

Reply:
Marking this as resolved, but please add some comments about why these instance arguments are here